content/en/blog/2024/target-allocator-troubleshooting/index.md

```diff
@@ -0,0 +1,481 @@
+---
+title: Tips for Troubleshooting the Target Allocator
+linkTitle: Target Allocator Troubleshooting
+date: 2024-06-17
+author: '[Adriana Villela](https://github.com/avillela) (ServiceNow)'
+canonical_url: https://adri-v.medium.com/de9eca2b78b4
+issue: 4699
+sig: OTel Operator
+cSpell:ignore: automagically bleh
+---
+
+![Looking up at the glass pyramid at the Louvre, as seen from the inside. Photo by Adriana Villela](louvre-pyramid.jpg)
+
+If you’ve enabled
+[Target Allocator](https://adri-v.medium.com/lets-learn-about-the-otel-operator-s-target-allocator-47a2b1f07562)
+service discovery on the
+[OTel Operator](https://adri-v.medium.com/list/opentelemetry-operator-0ee6378d630a),
+and your metrics aren’t getting picked up, then there are a few troubleshooting
+steps that you can take to help you understand what’s going on and to get things
+back on track. I put these together based on some of my own experience. May
+these help you on your own journey!
+
+## Troubleshooting Steps
+
+Before we start, be sure to check out
+[this repository](https://github.com/avillela/otel-target-allocator-talk),
+which, among other things, includes examples of configuring the
+`OpenTelemetryCollector` custom resource (CR) to use the Target Allocator’s
+service discovery functionality, along with examples of
+[`ServiceMonitor`](https://observability.thomasriley.co.uk/prometheus/configuring-prometheus/using-service-monitors/#:~:text=The%20ServiceMonitor%20is%20used%20to,build%20the%20required%20Prometheus%20configuration.)
+and
+[`PodMonitor`](https://prometheus-operator.dev/docs/user-guides/getting-started/#using-podmonitors)
+resource definitions.
+
+### 1- Did you actually deploy all of your resources to Kubernetes?
```

Comment:
This step has a different tone than the rest of post. You might consider editing it to make the post more cohesive.